mlflow/store/tracking/dbmodels/models.py

```diff
@@ -1361,3 +1363,220 @@ class SqlSpan(Base):
         ),  # For type-only and type+status filters
         Index("index_spans_experiment_id_duration", "experiment_id", "duration_ns"),
     )
+
+
+class SqlEvaluationDataset(Base):
+    """
+    DB model for evaluation datasets.
+    """
+
+    __tablename__ = "evaluation_datasets"
+
+    dataset_id = Column(String(36), primary_key=True)
+    """
+    Dataset ID: `String` (limit 36 characters).
+    *Primary Key* for ``evaluation_datasets`` table.
+    """
+
+    name = Column(String(255), nullable=False)
+    """
+    Dataset name: `String` (limit 255 characters). *Non null* in table schema.
+    """
+
+    tags = Column(JSON, nullable=True)
```

Comment:
In order to support the standard CRUD operations for tags, we externalized this in a separate table that is handled with a foreign key relationship. https://github.com/mlflow/mlflow/blob/cdb2bcbf3cd87fd2b7f25cfe75a9926a5dea6e8a/mlflow/store/tracking/dbmodels/models.py#L1344-L1348 

We want to keep this migration consistent, right?